mautrix_telegram/portal/matrix.py

```diff
@@ -369,6 +369,11 @@ async def _send_bridge_error(self, msg: str) -> None:
 
     async def handle_matrix_message(self, sender: 'u.User', content: MessageEventContent,
                                     event_id: EventID) -> None:
+        if self.bridge.is_blocked:
```

Comment:
Would just double check this doesn't break admin commands btw

Reply:
It does not, it seems (though it wasn't intentional). I'm wondering if it should?